Jenkinsfile

```diff
@@ -52,10 +54,12 @@ node('ubuntu-zion') {
       }
       gitHub = new GitHub(this, "${organization}/${gitHubRepository}", apiToken)
     }
-    stage('Update IQ Version') {
-      OsTools.runSafe(this, "git checkout ${branch}")
-      dockerFileLocations.each { updateServerVersion(it, nexusIqVersion, nexusIqSha) }
-      version = getShortVersion(nexusIqVersion)
+    if ((nexusIqVersion && nexusIqSha) && branch == 'master') {
```

Comment:
minor simplification to suggest here - use the same variable `env.releaseBuild_NAME` as above here to make it clear it's the same condition being applied

Reply:
> minor simplification to suggest here - use the same variable `env.releaseBuild_NAME` as above here to make it clear it's the same condition being applied

:+1:  1967f6f5